packages/ag-charts-website/src/content/menu/data.json

```diff
@@ -0,0 +1,352 @@
+                    { "title": "Themes", "path": "charts-themes" },
+                    { "title": "Overlays", "path": "charts-overlays" },
+                    { "title": "Background", "path": "charts-background" },
+                    {
+                        "title": "Background Image",
+                        "path": "charts-background-image",
+                        "isEnterprise": true
+                    }
+                ]
+            },
+            {
+                "title": "Axes",
+                "items": [
+                    { "title": "Axis Types", "path": "charts-axes-types" },
+                    { "title": "Axis Ticks", "path": "charts-axes-ticks" },
+                    { "title": "Axis Domain", "path": "charts-axes-domain" },
+                    { "title": "Axis Labels", "path": "charts-axes-labels" },
+                    { "title": "Grid Lines", "path": "charts-axes-grid-lines" },
+                    { "title": "Secondly Axes", "path": "charts-axes-secondary" }
+                ]
+            },
+            {
+                "title": "Features",
+                "items": [
+                    { "title": "Legend", "path": "charts-legend" },
+                    { "title": "Series Markers", "path": "charts-markers" },
+                    { "title": "Series Highlighting", "path": "charts-series-highlighting" },
+                    { "title": "Tooltips", "path": "charts-tooltips" },
+                    { "title": "Formatters", "path": "charts-formatters" },
+                    { "title": "Cross lines", "path": "charts-axes-cross-lines" },
+                    { "title": "Navigator", "path": "charts-navigator" },
+                    {
+                        "title": "Crosshairs",
+                        "path": "charts-axes-crosshairs",
+                        "isEnterprise": true
+                    },
+                    {
+                        "title": "Context Menu",
+                        "path": "charts-context-menu",
+                        "isEnterprise": true
+                    },
+                    {
+                        "title": "Zoom",
+                        "path": "charts-zoom",
+                        "isEnterprise": true
+                    },
+                    {
+                        "title": "Animation",
+                        "path": "charts-animation",
+                        "isEnterprise": true
+                    }
+                ]
+            },
+            {
+                "title": "AG Grid Integrated Charts",
+                "url": "https://ag-grid.com/javascript-data-grid/integrated-charts/",
+                "isEnterprise": true
+            }
+        ]
+    },
+    "charts": {
+        "items": [
+            {
+                "title": "Line",
+                "icon": "chartsLine",
+                "items": [
+                    {
+                        "title": "Overview",
```

Comment:
`"title": "Overview"` is repeated across all 2nd-level items, why is that?

Reply:
They are placeholders for now, and may be generated in future. It's easier to change as items in the menu data json for now